Composer/plugins/samples/assets/projects/ToDoBotWithLuisSample/.gitignore

```diff
@@ -1,9 +0,0 @@
-# NuGet Packages
-*.nupkg
-# The packages folder can be ignored because of Package Restore
-**/[Pp]ackages/*
-# except build/, which is used as an MSBuild target.
-!**/[Pp]ackages/build/
-
-!Properties/launchSettings.json
-!packages/packages.json
```

Comment:
Are we sure these should not also get ignored? The new `.gitignore` only ignores `appsettings.json`.

Reply:
I think those folder were stale, those code added in 7 month ago. I remember we had the C# runtime copy with sample bot previously, and it's relevant with those structure only.